packages/storefront-theme/assets/css/product-filters.css

```
@@ -0,0 +1,171 @@
+
+.bt-product-filters__toolbar {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  gap: 0.5rem;
+}
+
+.bt-product-filters__toggle,
+.bt-product-filters__actions button {
+  min-height: 44px;
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)